tests/Unit/ValueObject/Result/SearchResultsTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+
+namespace AsyncBot\Plugin\ImdbTest\Unit\ValueObject\Result;
+
+use AsyncBot\Plugin\Imdb\ValueObject\Result\SearchResult;
+use AsyncBot\Plugin\Imdb\ValueObject\Result\SearchResults;
+use AsyncBot\Plugin\Imdb\ValueObject\Result\Type;
+use PHPUnit\Framework\TestCase;
+
+class SearchResultsTest extends TestCase
```

Comment:
This is missing a test case when there are no search results https://coveralls.io/builds/26438922/source?filename=src/ValueObject/Result/SearchResults.php#L18